src/server.ts

```diff
@@ -29,7 +28,7 @@ bot.api.config.use(
 // middleware
 bot.use(
   session({
-    initial: () => (Promise.resolve({} as any)),
+    initial: () => ({}),
```

Comment:
what it does?

Reply:
initialize data for session

Reply:
yeah i just wonder what caused you to change it, is it gave any errors? or smth like this or it's better approach or etc..

Reply:
oldin o’zi shunaqa turudi

Reply:
aaa ok